src/ray/protobuf/autoscaler.proto

```diff
@@ -0,0 +1,172 @@
+/*
+This file defines an experimental interface designed for Ray
+autoscalers.
+
+It is currently unstable and under active development, but it is
+intended to become stable and forwards-compatible in the future.
+*/
+
+
+syntax = "proto3";
+option cc_enable_arenas = true;
+package ray.rpc;
+
+message ResourceBundle {
+  // The fixed point resource size of the resource (granularity: 1/10000).
+  map<string, int64> resources = 1;
```

Comment:
does unit resource instances apply here?

Reply:
No, this is a bug in the current autoscaler too.  Actually, I think this field is fine, (you can't do `num_gpus=[0.2, 0.3]`), but for the node resource usage, do you think we should make them lists of available? (it _feels_ kinda bad, but I don't see a better option if we want to fix that bug).

Reply:
Can you say more about what the bug is?